ui/src/components/DashboardServices.js

```diff
@@ -0,0 +1,322 @@
+// @flow
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+import { Tooltip } from '@scality/core-ui';
+import {
+  spacing,
+  fontSize,
+  fontWeight,
+} from '@scality/core-ui/dist/style/theme';
+
+import type { Alert } from '../services/alertUtils';
+import { PageSubtitle } from '../components/style/CommonLayoutStyle';
+import {
+  useAlertLibrary,
+  useHighestSeverityAlerts,
+  highestAlertToStatus,
+} from '../containers/AlertProvider';
+import CircleStatus from './CircleStatus';
```

Comment:
I thought we already move it core-ui, but apparently not.. ;) 
Not the scope for this PR will do it in another PR.